core/templates/pages/terms-page/terms-page.mainpage.html

```diff
@@ -321,5 +306,19 @@ <h2>Changelog</h2>
       </style>
     </oppia-root>
     @load('pages/footer_js_libs.html')
+    <script>
+      // This scrolls the page to the anchor clicked on in the quick links menu.
+      $('a[href*=\\#]:not([href=\\#])').click(function() {
+        if (location.pathname.replace(/^\//, '') == this.pathname.replace(/^\//, '')) {
+          var target = $('[name=' + this.hash.slice(1) +']');
+          if (target.length) {
+            $('html,body').animate({
+              scrollTop: target.offset().top
+            }, 1000);
+            return false;
+          }
+        }
+      });
```

Comment:
Why is this moved, also it is possible to replace this by Angular? We want to limit the direct usage of jQuery as much as possible.

Reply:
> Why is this moved

It uses jQuery and jQuery is now loaded in the webpack bundle, so, it needs to be after the footer_libs import.

> is it possible to replace this by Angular?

This page currently has no controller. It's just a `mainpage.html`. 

Also, I think the smooth scroll doesn't work. You can see it here -> https://www.oppia.org/terms

Reply:
Could you please introduce a controller then?

Reply:
Or somehow manage it so that there is functionality similar to this piece of code.

Reply:
done. added an angular component. Also fixed the smooth scroll :) 

before
![before (1)](https://user-images.githubusercontent.com/36989112/89416556-0f73fc80-d74b-11ea-9190-87126505024b.gif)

after
![after (1)](https://user-images.githubusercontent.com/36989112/89416660-3a5e5080-d74b-11ea-8795-ec22357dc6fb.gif)

Reply:
Thanks! It looks great now!